CMakeLists.txt

```diff
@@ -468,6 +478,7 @@ set (cxx-sources
         plain_server.cpp
         poll.cpp
         poller_base.cpp
+		pollset.cpp
```

Comment:
In the next commit could you also please fix the indentation? It's spaces only, no tabs